install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```diff
@@ -25,6 +26,7 @@ spec:
   selector:
     matchLabels:
       k8s-app: cilium
+      app.kubernetes.io/part-of: cilium
```

Comment:
@gandro To your point about service selectors, we probably want to extend that to template selectors?

Reply:
@tommyp1ckles I think when we labels all cilium resources with `app.kubernetes.io/name` or `app.kubernetes.io/component`, They should also be used for `selector` or `matchLabels`, Just like `k8s-app`, And `app.kubernetes.io/part-of` is only for label.

Reply:
I gave this some more thought, and yes I actually agree that using both labels (`name` and `component`) in the selectors is better. So please ignore my previous request.